docs/reference/search.asciidoc

```diff
@@ -39,6 +39,12 @@ exception of the <<search-explain,explain API>>.
 * <<multi-search-template>>
 * <<render-search-template-api>>
 
+ifdef::permanently-unreleased-branch[]
+[discrete]
+[[geo-search-apis]]
+=== Geospatial search
+* <<vector-tile-api>>
+endif::[]
```

Comment:
These `ifdef`/`endif` tags ensure these docs are only available on 7.x and master. These tags will hide the docs on 7.15 and other branches until removed. This is common practice for docs under a feature flag.